2_Growth-Maturity-Decline.md

```diff
@@ -27,6 +27,7 @@
  --- | ---
  Total contributors | The total number of contributors over time on any platform.
  Total new contributors | The total number of new contributors over time on any platform.
+ Total new (sub)projects | The total number of new (sub)projects created over time

```

Comment:
I like this metric, but I don't think there is a need for "new" here. Just make it Total (sub)projects

Reply:
That makes sense.  So the description would be "The total number of (sub)projects over time"?